docs/pre-commit-hooks.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "pre-commit hooks"
+draft: false
+type: docs
+layout: single
+
+menu:
+  docs:
+    weight: 120
+---
+
+# pre-commit hooks
+
+`pre-commit` is a tool which manages running of
+[git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) scripts.
+See the official documentation for more information: [pre-commit.com](https://pre-commit.com/)
```

Comment:
Sorry about all these nitpicks -- there's lots of passive voice and odd grammar that I'd like to correct before merge.

```suggestion
pre-commit is a framework for building and running
[git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks).
See the official documentation for more information: [pre-commit.com](https://pre-commit.com/)
```